app/scripts/modules/core/delivery/triggers/nextRun.component.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const angular = require('angular');
+
+let later = require('later');
+
+module.exports = angular
+  .module('spinnaker.core.deliver.triggers.nextRun', [
+    require('../../utils/moment'),
+    require('../../config/settings'),
+  ])
+  .component('nextRunTag', {
+    bindings: {
+      pipeline: '<'
+    },
+    controller: function (momentService, settings) {
+
+      function setScheduled() {
+        if (!this.pipeline) {
+          return;
+        }
+        let [cron] = this.pipeline.triggers.filter(t => t.type === 'cron' && t.enabled);
+        if (cron) {
```

Comment:
Ugh, it does now, although shame on whoever configured multiple cron triggers for their pipeline.